specification/window.browser.html

```diff
@@ -0,0 +1,1673 @@
+   <p></p>
+  </div>
+  <div data-fill-with="at-risk"></div>
+  <nav data-fill-with="table-of-contents" id="toc">
+   <h2 class="no-num no-toc no-ref" id="contents">Table of Contents</h2>
+   <ol class="toc" role="directory">
+    <li>
+     <a href="#w3c-conformance"><span class="secno"></span> <span class="content">Conformance</span></a>
+     <ol class="toc">
+      <li><a href="#w3c-conventions"><span class="secno"></span> <span class="content">Document conventions</span></a>
+     </ol>
+    <li>
+     <a href="#index"><span class="secno"></span> <span class="content">Index</span></a>
+     <ol class="toc">
+      <li><a href="#index-defined-here"><span class="secno"></span> <span class="content">Terms defined by this specification</span></a>
+      <li><a href="#index-defined-elsewhere"><span class="secno"></span> <span class="content">Terms defined by reference</span></a>
+     </ol>
+    <li>
+     <a href="#references"><span class="secno"></span> <span class="content">References</span></a>
+     <ol class="toc">
+      <li><a href="#normative"><span class="secno"></span> <span class="content">Normative References</span></a>
+     </ol>
+    <li><a href="#idl-index"><span class="secno"></span> <span class="content">IDL Index</span></a>
+   </ol>
+  </nav>
+  <main>
+   <section class="non-normative">
+    <h3 class="heading settled" id="introduction"><span class="content">Introduction</span></h3>
+     <em>This section is non-normative.</em> 
+    <p>Modern WebExtensions are an evolution of a system originally introduced in
+  Chrome version 3. In that system, any API that was added was put under the
+ `chrome` namespace. When WebExtensions were adopted outside of Chromium, other
+  vendors used the more neutral `browser` namespace. That `browser` is now
+  considered to canonically correct namespace for all WebExtension APIs.
+  Some interactions with WebExtensions (such as external messaging from content
+  scripts to background processes) necessitate the exposure of this namespace
+  to the web.
+  Given that, this document defines `window.browser` as reserved for the use of
+  APIs designed to interact with WebExtensions. The specifics of what
+  APIs within `window.browser` are at present intentionally undefined, and are
+  left up to User Agent implementation.</p>
+   </section>
+   <section>
+    <h3 class="heading settled" id="window-interface"><span class="content"> <a class="idl-code" data-link-type="attribute" href="https://compat.spec.whatwg.org/#dom-window-orientation" id="ref-for-dom-window-orientation"><code>window.browser</code></a> API </span></h3>
+    <p><code class="idl"><a data-link-type="idl" href="#dom-window-browser" id="ref-for-dom-window-browser">browser</a></code> is UA defined attribute exposed on <code class="idl"><a data-link-type="idl" href="https://fetch.spec.whatwg.org/#dom-requestinit-window" id="ref-for-dom-requestinit-window">window</a></code>. When implemented,
```

Comment:
`window` should should likely reference `https://html.spec.whatwg.org/#window` instead of the Fetch spec.